spec/high_voltage/configuration_spec.rb

```diff
@@ -61,7 +74,20 @@
     end
   end
 
-  describe '#page_caching=' do
+  describe '#action_caching_layout=false' do
+    it 'does not display a deprecation warning' do
+      allow(ActiveSupport::Deprecation).to receive(:warn)
+
+      HighVoltage.configure do |config|
+        config.action_caching_layout = false
+      end
+
+      expect(ActiveSupport::Deprecation).not_to have_received(:warn)
+                                                .with(HighVoltage::Configuration::CACHING_DEPRECATION_WARNING)
+    end
+  end
+
+  describe '#page_caching=true' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.